source/codegen/common_helpers.py

```diff
@@ -33,9 +33,9 @@ def is_input_parameter(parameter):
     return "in" in parameter["direction"]
 
 
-def is_pointer_parameter(parameter):
-    """Whether the parameter is a pointer parameter."""
-    return is_output_parameter(parameter) or parameter.get("pointer", False)
+def levels_of_pointer_indirection(parameter: dict) -> int:
```

Comment:
Note: Taken directly from https://github.com/ni/grpc-device/pull/607, will go away after that goes in and this branch is rebased.